project/Dependencies.scala

```diff
@@ -24,6 +24,8 @@ object Dependencies {
     .withRank(KeyRanks.Invisible) // avoid 'unused key' warning
 
   val junitVersion = "4.13.2"
+  val junit5Version = "5.10.0-M1"
```

Comment:
can you use an actual release version instead of a pre-release milestone?

Reply:
yes, i changed it to 5.9.3